code/game/objects/structures/grille.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "grille"
 	density = TRUE
 	anchored = TRUE
+	bound_height = 30
+	bound_width = 26
```

Comment:
So throwing items doesn't hit the grille underneath the window

Reply:
I was specifically trying to avoid solutions like this in regards to multiple things getting bumped because it will very quickly end up janky. Multiple things get bumped when they have the same boundary and that's fine.

Reply:
I’ll revert this change then, it might not be an issue anymore anyway because I later reduced the bounds of most throwing items

Reply:
It's still an issue but I reverted it anyway